buildSrc/version.properties

```diff
@@ -21,6 +21,7 @@ woodstox          = 6.4.0
 kotlin            = 1.7.10
 antlr4            = 4.11.1
 guava             = 31.1-jre
+protobuf          = 3.22.2
```

Comment:
Apologies @saratvemulapalli , non-issue but I've just realized a minor update for protobuf is ongoing (https://github.com/opensearch-project/OpenSearch/pull/7261/files), not a big deal, I could rebase it after